src/nimble.nim

```diff
@@ -1028,6 +1028,10 @@ proc doAction(options: Options) =
     let (_, pkgInfo) = install(options.action.packages, options)
     if options.action.packages.len == 0:
       nimScriptHint(pkgInfo)
+    if pkgInfo.foreignDeps.len > 0:
+      echo("To finish the installation, run: ")
```

Comment:
``echo`` should no longer be used in Nimble to display information. Use the new ``display`` proc:

```nim
display("Hint:", "To finish the installation, run:", Warning, HighPriority)
```